src/main/kotlin/ch/yvu/rxpect/mockito/MockitoHelpers.kt

```diff
@@ -4,16 +4,11 @@ import org.mockito.Mockito
 import org.mockito.invocation.Invocation
 import org.mockito.stubbing.OngoingStubbing
 import org.mockito.stubbing.Stubbing
-import java.util.TreeSet
 
 object MockitoHelpers {
     fun <T> extractLastInvocation(ongoingStubbing: OngoingStubbing<T>): Invocation =
         getAllStubbings(ongoingStubbing.getMock()).last().invocation
 
-    private fun getAllStubbings(mock: Any): List<Stubbing> {
-        // We use the internal information, that Mockito tracks stubbings ordered in a TreeSet
```

Comment:
The comment should not be removed but updated.
We still assume that the provided collection from stubbings is ordered (the signature is a Collection - only the implementation says it is a LinkedList) and the last element corresponds to the last invocation.